src/Oceananigans.jl

```diff
@@ -143,7 +143,7 @@ import Base:
 # (libnetcdf) are installed on the machine
 
 "Boolean denoting whether NCDatasets.jl can be loaded on machine."
-const hasnetcdf = @static (Sys.islinux() && Sys.ARCH == :x86_64) ? true : false
+const hasnetcdf = @static (Sys.islinux() && Sys.ARCH == :powerpc64le) ? false : true

```

Comment:
I still don't grasp the problem with try-catch. Does it not work? It's better.

Reply:
as I said,
> Actually, try-catch won't cut the mustard for satori (=won't do the job)

I don't know why though :)

Reply:
Note that people who manage to install NetCDF on powerpc will _not_ be able to use NetCDF with Oceananigans despite all their hardwork! They'll have to change this source code. That shows you this is actually wrong, in principle. The problem is not at all that NetCDF doesn't work on powerpc. The problem is that binaries are not _provided automagically by Julia_ because NetCDF_jll doesn't work for powerpc.

Reply:
It doesn't work. I added it and then called `using Oceananigans` and I was getting errors related with `libnetcdf` not existing, etc.

Reply:
OK, I agree!
I just can't figure out a way to implement this so that it spits no errors on a machine that does not have the NetCDF_jll.

`try-catch` is a great suggestion but unfortunately doesn't work :(

Reply:
why doesn't it work? What's the error?

Reply:
perhaps we need to include the `try-catch` inside the `__init__()`?

Reply:
Maybe a test with NetCDF_jll directly rather than NCDatasets?

Reply:
OK, let me do it again

Reply:
```Julia
navidcon@satori-login-001:~/Oceananigans.jl$ julia --project
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.8.2 (2022-09-29)
 _/ |\__'_|_|_|\__'_|  |
|__/                   |

julia> using Oceananigans
[ Info: Precompiling Oceananigans [9e8cae18-63c1-5223-a75c-80ca9d6e9a09]
ERROR: InitError: UndefVarError: libnetcdf not defined
Stacktrace:
  [1] nc_inq_libvers
    @ ~/.julia/packages/NCDatasets/ipGBH/src/netcdf_c.jl:242 [inlined]
  [2] netcdf_version()
    @ NCDatasets ~/.julia/packages/NCDatasets/ipGBH/src/netcdf_c.jl:2157
  [3] init_certificate_authority()
    @ NCDatasets ~/.julia/packages/NCDatasets/ipGBH/src/netcdf_c.jl:2170
  [4] __init__()
    @ NCDatasets ~/.julia/packages/NCDatasets/ipGBH/src/NCDatasets.jl:33
  [5] _include_from_serialized(pkg::Base.PkgId, path::String, depmods::Vector{Any})
    @ Base ./loading.jl:831
  [6] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String, build_id::UInt64)
    @ Base ./loading.jl:1039
  [7] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64)
    @ Base ./loading.jl:898
  [8] _tryrequire_from_serialized(pkg::Base.PkgId, path::String)
    @ Base ./loading.jl:971
  [9] _require(pkg::Base.PkgId)
    @ Base ./loading.jl:1347
 [10] _require_prelocked(uuidkey::Base.PkgId)
    @ Base ./loading.jl:1200
 [11] macro expansion
    @ ./loading.jl:1180 [inlined]
 [12] macro expansion
    @ ./lock.jl:223 [inlined]
 [13] require(into::Module, mod::Symbol)
    @ Base ./loading.jl:1144
during initialization of module NCDatasets

Julia>
```